packages/nextjs/app/builders/0xcefb233406133224AE9b19444d836F6a33593F13/page.tsx

```diff
@@ -0,0 +1,68 @@
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const ProfilePage: NextPage = () => {
+  return (
+    <>
+      <div className="flex flex-col gap-y-6 lg:gap-y-8 py-8 lg:py-12 justify-center items-center">
+        <div
+          className="bg-base-100 border-base-300 border shadow-md shadow-secondary rounded-3xl px-6 lg:px-8 mb-6 space-y-1 py-4"
+          style={{ width: "80%" }}
```

Comment:
Nit: Actually already there is tailwind class present for `80%`  `w-4/5`.  And also sometimes if tailwind doesn't provide the class for that value out of box you can use the bracket syntax to use any arbitary value eg: `w-[84%]`. 

here are the [docs for arbitary values](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values).

Reply:
Thx!